src/commands/logging/LogKey.ts

```diff
@@ -81,6 +104,21 @@ export class LogKey extends BaseCommand {
         await MongoManager.addIdNameToIdNameCollection(resMember);
         await MongoManager.getOrCreateUserDoc(resMember.id);
 
+        const userToLogFor = resMember.id === ctx.user.id ? "yourself" : resMember.toString();
+
+        // Check if there is already a key pre-selected
+        if (preselectedKey) {
+            const preselectedDungeon = MAPPED_AFK_CHECK_REACTIONS[preselectedKey];
+            await LoggerManager.logKeyUse(resMember, preselectedKey, keys);
+
+            await ctx.interaction.reply({
+                ephemeral: true,
```

Comment:
Yeah, that's fine with me. The reason why I don't usually do ephemeral is because sometimes we have to defer the response to the interaction, which then makes ephemeral responses impossible. But, if you're careful enough, then it shouldn't be an issue.

Reply:
Oh, didn't know that. In that case, we can do that then.